### DISCUSSION TITLE
I want to remove the Export button from IP Addresses and Prefixes.

### DISCUSSION THREAD

**Discussion by kgw0820** [netbox-community/netbox#14800] [Q&A]
I want to remove the export button from IP Addresses and Prefixes.
![image](https://github.com/netbox-community/netbox/assets/141695004/3be3bff6-e1aa-4321-af91-c53021987371)

It is intended to provide view permission to general users and to prevent information leakage due to export.
Of course, information leakage is possible through the screen viewed by the user, but the act of taking the entire data into a csv file is unacceptable.

So what should I do to get rid of the export button? It would be nice if there was a simple function, but if I need to modify the source code, I need detailed help. I am a network engineer with no development knowledge.

**Comment by alehaa** [comment]
I don't think that's possible as exporting the data uses the same permissions as the API read access or displaying the HTML table. Of course you can tweak the django template to hide the button, but you would also have to remove the associated routes etc. In my opinion this is just security by obscurity and primarily a layer 8 problem.
